quartz/components/Explorer.tsx

```diff
@@ -26,49 +27,58 @@ const defaultOptions = {
 } satisfies Options
 
 export default ((userOpts?: Partial<Options>) => {
-  function Explorer({ allFiles, displayClass, fileData }: QuartzComponentProps) {
-    // Parse config
-    const opts: Options = { ...defaultOptions, ...userOpts }
+  // Parse config
+  const opts: Options = { ...defaultOptions, ...userOpts }
 
-    // Construct tree from allFiles
-    const fileTree = new FileNode("")
-    allFiles.forEach((file) => fileTree.add(file, 1))
+  // memoized
+  var fileTree: FileNode
+  var jsonTree: string
```

Comment:
do we need `var` over `let` here?

Reply:
ah good node, i'll fix it lol i always forget whether `var` or `let` has bigger scope